back/config/graphql/types/Buying/mutations.yaml

```
@@ -0,0 +1,35 @@
+BuyingMutations:
+  type: object
+  config:
+    fields:
+
+      create:
+        type: Buying
+        resolve: "@=mutation('Buying.create', args)"
+        access: "@=isAuthenticated()"
+        description: Ajouter une nouvelle transaction d'achat
+        args:
+          payload:
+            type: BuyingPayload!
+
+      update:
+        type: Buying
+        resolve: "@=mutation('Buying.update', args['uid'], args)"
+        access: "@=isAuthenticated()"
+        description: Modifier une transaction d'achat
+        args:
+          uid:
+            type: ULID!
+            description: 'Unique identifier for the buying transaction'
+          payload:
+            type: BuyingPayload!
+
+      delete:
+        type: ULID!
```

```suggestion
        type: Boolean!
```
Je t'explique un peu plus bas pourquoi 😉